ci_handbook/index.qmd

```diff
@@ -8,8 +8,19 @@ format:
     comments:
       utterances:
          repo: eco4cast/efi-ci-workshop-2024
+listing: 
+  contents: 
```

Comment:
In the meeting we talked about an 'Examples' section, or would that be included under workflows?

Reply:
Oh yeah, I think that would be it's own section / article. I added a new placeholder for that. Thanks!